src/internal/operators/dematerialize.ts

```diff
@@ -25,26 +25,26 @@ import { OperatorSubscriber } from './OperatorSubscriber';
  * Convert an Observable of Notifications to an actual Observable
  *
  * ```ts
- * import { of } from 'rxjs';
- * import { dematerialize } from 'rxjs/operators';
+ * import { NextNotification, ErrorNotification, of, dematerialize } from 'rxjs';
  *
- * const notifA = { kind: 'N', value: 'A' };
- * const notifB = { kind: 'N', value: 'B' };
- * const notifE = { kind: 'E', error: new TypeError('x.toUpperCase is not a function') }
+ * const notifA = <NextNotification<string>>{ kind: 'N', value: 'A' };
```

Comment:
Please prefer `const notifA: NextNotification<string> = ` or `{} as NextNotification<string>`. We're trying to move away from bracket casts.

Reply:
This is a "nit". We can fix it later.

Reply:
Sure, I'll fix in another PR.